src/main/java/app/repository/comment/CommentRepository.java

```diff
@@ -0,0 +1,15 @@
+package app.repository.comment;
+
+import app.model.Comment;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface CommentRepository extends JpaRepository<Comment, Long> {
+    @Query("SELECT c FROM Comment c WHERE c.user.id = :userId")
+    List<Comment> findAllByUserId(Long userId, Pageable pageable);
```

Comment:
```
@Query("SELECT c FROM Comment c JOIN FETCH c.user WHERE c.user.id = :userId")
List<Comment> findAllByUserId(@Param("userId") Long userId, Pageable pageable);
```
or even can it be replaced with JpaRepository method 

`List<Comment> findByUserId(Long userId, Pageable pageable);` ?